app/dialogs.js

```diff
@@ -140,9 +155,14 @@ var appDialogs = {
         dialog.showMessageBox(win, {
             title: "Images Copied",
             message: "Document images have been copied in the '" + path + "' directory.",
-            buttons: ['OK'],
+            buttons: ["OK"],
             noLink: true
         });
+    },
+
+    warnFileDeleted: function (path, callback) {
```

Comment:
Makes sense.
I actually hesitated between these two behaviors.

Sublime Text simply changes the document state to "dirty", but Notepad++ does what you are describing.